resources/templates/docker-compose.yaml.tpl

```diff
@@ -3,11 +3,14 @@ services:
   openvpn:
     cap_add:
      - NET_ADMIN
-    image: kylemanna/openvpn:2.4
+    image: public.ecr.aws/n5x5g3h7/nimbux911/dockovpn:2.6.8-config.0.1.0
     container_name: openvpn
     ports:
      - "1194:1194/udp"
     restart: always
+    environment:
+     - DEBUG=1
```

Comment:
Thanks for pointing out.. i don't think that parameterize too many variables one by one makes sense in this case, if anything it'd be better to play around with the tpl syntax to inject variables more dynamically form the TF module instance.

In this case "EASYRSA_BATCH=1" is hardcoded since it's necessary for the user data to run properly since the initpli scripts needs it to avoid any user prompt halting the installation.

Reply:
For now i'm removing the hardcoded DEBUG=1

Reply:
Great, thanks!